tests/footballEasterEggs.test.ts

```
@@ -0,0 +1,45 @@
+    expect(isMinute67({ ...baseGoal, minute: '90+67' })).toBe(false);
+    expect(isMinute67({ ...baseGoal, minute: '66' })).toBe(false);
+  });
+
+  test('isScore71 matches either orientation', () => {
+    expect(isScore71(7, 1)).toBe(true);
+    expect(isScore71(1, 7)).toBe(true);
+    expect(isScore71(7, 0)).toBe(false);
+  });
+
+  test('getGoalFollowUpContent stacks easter eggs', () => {
+    const goal: GoalEvent = { ...baseGoal, home: 7, away: 1, minute: '67' };
```

ESLint

[error] Expected a line break after this opening brace.

(object-curly-newline)